### PR TITLE
Add PUI info banner to CUI

### LINKED ISSUE #6554: Add links to the new ui in the old ui like `try this out in the new ui` @wolflu05

### PR DESCRIPTION
**Pull request by matmair**
This adds callouts and links to PUI at several points in CUI. Can be disabled with a customize setting.
grafik
grafik
grafik

Closes #6554

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
Can we use a different wording? Do you mean `interstitial`? Perhaps "banner" would be better?

**Comment by matmair** [comment]
Sure, not married to any wording. I have changed all occurrences. My english is not the yellow of the egg sometimes https://www.youtube.com/shorts/0jAGDb5pXzs ;-)

File: docs/docs/start/config.md [hunk 1]
```diff
@@ -324,6 +324,7 @@ The logo and custom messages can be changed/set:
 | INVENTREE_CUSTOM_SPLASH | customize.splash | Path to custom splash screen in the static files directory | *Not specified* |
 | INVENTREE_CUSTOMIZE | customize.login_message | Custom message for login page | *Not specified* |
 | INVENTREE_CUSTOMIZE | customize.navbar_message | Custom message for navbar | *Not specified* |
+| INVENTREE_CUSTOMIZE | customize.hide_pui_banner | Disable PUI intrestial | False |
```
Comment (SchrodingersGat) [inline docs/docs/start/config.md:327]: Please change `intrestial` to `banner` here too :)

Comment (matmair) [reply docs/docs/start/config.md:327]: done!

**Comment by SchrodingersGat** [comment]
Cheers @matmair for the yellow of the egg

**Comment by HannesMorre** [comment]
Is there a way to go back to CUI from PUI? 🙃

**Comment by wolflu05** [comment]
> Is there a way to go back to CUI from PUI? 🙃

@HannesMorre  Yes, just remove the path from the url. CUI is currently accessible by `https://your.instance.com/` while the link in the banner just navigates you to `https://your.instance.com/platform`. So all you need is to navigate back to `https://your.instance.com/`

**Pull request merged**